helloworld/pom.xml

```diff
@@ -235,6 +235,50 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>integration-testing-with-provisioned-server</id>
```

Comment:
We also need the `maven-failsafe-plugin` defined and a way to define the `server.host` by default IMO. This won't work unless you explicitly set that property if used with the `provisioned-server` profile.